src/scripts/main.js

```diff
@@ -3,6 +3,22 @@
 const people = require('./lib/people');
 
 // eslint-disable-next-line no-console
-console.log(people); // you can remove it
 
-// write your code here
+function addRow(arg) {
+  const table = document.querySelector('table');
+
+  for (const person of people) {
```

Comment:
Replace the for loop with a forEach loop for better readability and to follow functional programming practices.